src/scenes/Login/LogInSignUp.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
+import Login from '../Login/components/Login'
+import SignUp from './components/SignUp'
+
+const LoginSignUp = () => {
+  return (<Router>
+    <div className="App">
+      <nav className="navbar navbar-expand-lg navbar-light fixed-top">
+        <div className="container">
+          <Link className="navbar-brand" to={'/sign-in'}></Link>
+          <div className="collapse navbar-collapse" id="navbarTogglerDemo02">
+            <ul className="navbar-nav ml-auto">
+              <li className="nav-item">
+                <Link className="nav-link" to={'/sign-in'}>Login</Link>
+              </li>
+              <li className="nav-item">
+                <Link className="nav-link" to={'/sign-up'}>Sign up</Link>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
```

Comment:
Never see a login page contain a navbar
A login page should only contain the login form
![image](https://user-images.githubusercontent.com/14868839/122050921-3476d880-ce0e-11eb-90e7-89bfa5959812.png)

Reply:
done